app/controllers/Checkout.scala

```diff
@@ -57,11 +57,36 @@ object Checkout extends Controller with LazyLogging with ActivityTracking with C
   def getEmptySubscriptionsForm(promoCode: Option[PromoCode])(implicit res: TouchpointBackend.Resolution) =
     SubscriptionsForm.subsForm.bind(Map("promoCode" -> promoCode.fold("")(_.get)))
 
+  def getCheckoutRouteForProductFamily(countryGroup: CountryGroup, promoCode: Option[PromoCode], product: ProductFamily) = {
+    product match {
+      case Paper => routes.Checkout.renderPaperCheckout(countryGroup, promoCode)
+      case Digipack => routes.Checkout.renderDigipackCheckout(countryGroup, promoCode)
+      case _ => routes.Homepage.index()
+    }
+  }
+
+  def renderProductCheckout(countryGroup: CountryGroup, promoCode: Option[PromoCode], product: ProductFamily) = AuthorisedTester.async { req =>
```

Comment:
am I right in thinking that the intention is to replace renderPaper / renderDigipack with calls to this when we release paper & sort out analytics?